install.sh

```diff
@@ -40,7 +40,7 @@ esac
 
 # This will allow things to work without sudo
 crew_folders="bin cache doc docbook etc include lib lib$LIB_SUFFIX libexec man sbin share tmp var"
-for folder in $crew_folders ; do sudo chown -R "$(id -u)":"$(id -g)" "${CREW_PREFIX}"/"$folder" ; done
+for folder in $crew_folders ; do sudo chown -R "$(id -u)":"$(id -g)" "${CREW_PREFIX}"/"$folder" || true ; done
```

Comment:
This may ignore any error other than `no such directory` in `chown`,  why not check the dir exist before `chown`?
```shell
cd ${CREW_PREFIX}
if [ -d ${folder} ] || [ -L ${folder} ]; then sudo chown -R $(id -u):$(id -g) ${folder}; fi
```

Reply:
This is a much better approach. Thanks!

Reply:
How do you feel about the new way? Testing it now...

Reply:
Looks good to me :) We can also simplify the `[ -d ${folder} ] || [ -L ${folder} ]` part to `[ -d ${folder} -o -L ${folder} ]`